viewer/src/examples/threejs/sector-with-pointcloud.ts

```diff
@@ -56,16 +56,16 @@ async function main() {
   camera.updateMatrixWorld();
 
   const clock = new THREE.Clock();
-  const render = () => {
+  const render = async () => {
     requestAnimationFrame(render);
```

Comment:
How does an async callback to requestAnimationFrame() work?

Reply:
We should probably wait with requesting another animation frame until the end of the render function - I do not think it is necessary, but should improve sanity a bit.